docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.5'
```

Comment:
why need to include this file?

Reply:
> hi, why needs to include this file in the repo? I think we can keep this in document since it's only for internvl we can add a section of how to deploy internvl using docker in this document https://github.com/InternLM/lmdeploy/blob/main/docs/en/multi_modal/internvl.md https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/multi_modal/internvl.md

i will add some content in docs about use  compose in next week,but i suggest to keep a docker-compose.yml in repo,it's helpful to people who use lmdeploy first without finish reading docs,and make lmdeploy easy to use;more users's participated will find more potential bug,it will save core developer's time to deal with more important question

Reply:
> hi, why needs to include this file in the repo? I think we can keep this in document since it's only for internvl we can add a section of how to deploy internvl using docker in this document https://github.com/InternLM/lmdeploy/blob/main/docs/en/multi_modal/internvl.md https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/multi_modal/internvl.md

i will move it to [internvl.md](https://github.com/InternLM/lmdeploy/blob/main/docs/en/multi_modal/internvl.md) next week,but i think we need a current compose in repo to make lmdeploy easy to use, some people use lmdeploy first without finish reading docs;